Marlin/Configuration.h

```diff
@@ -1361,6 +1361,7 @@
     #define MESH_TEST_HOTEND_TEMP  205    // (°C) Default nozzle temperature for the G26 Mesh Validation Tool.
     #define MESH_TEST_BED_TEMP      60    // (°C) Default bed temperature for the G26 Mesh Validation Tool.
     #define G26_XY_FEEDRATE         20    // (mm/s) Feedrate for XY Moves for the G26 Mesh Validation Tool.
+    #define G26_XY_FEEDRATE_TRAVEL  100   // (mm/s) Feedrate for XY Moves without extrusion for the G26 Mesh Validation Tool
```

Comment:
The travel value is still off by one.

Reply:
By the way, your change regarding the decimal point alignment is done in the new (needed) PR https://github.com/MarlinFirmware/Marlin/pull/20879 in commit https://github.com/MarlinFirmware/Marlin/pull/20879/commits/5d930a35753b399ba5234cbb19a77f29dde10858 - I finally understood what you wanted from me ;) I hope that is ok for you?

Reply:
All ok, thank you.